diagnostic.md

```diff
@@ -40,25 +54,30 @@ like to show all movies favorited by a profile on
 `http://localhost:3000/profiles/1`
 
   ```md
-    # < Your Response Here >
+A serializer takes the data from the request and converts it to JSON as a response.
   ```
 
   ```rb
   class ProfileSerializer < ActiveModel::Serializer
+  attributes :given_name, :surname, :email, :movies
+
+  def movies
+    object.movies.pluck(1)
+  end
   end
   ```
 
 1.  What would the command be to _scaffold_ out a **join table** for Favorites from
 the above `Movies` and `Profiles`.
 
   ```sh
-    # < Your Response Here >
+bin/rails destroy scaffold Favorites
```

Comment:
This will destroy an existing scaffold.  We want to create one.

```
bin/rails generate scaffold favorites movie:references profile:references
```